cards/card_component.js

```diff
@@ -9,6 +9,11 @@ BaseCard["{{componentName}}"] = class extends ANSWERS.Component {
 
     this.verticalKey = data.verticalKey;
     this.result = data.result || {};
+
+    this.globalOptions = this.analyticsReporter && this.analyticsReporter._globalOptions;
+    this.experienceKey = this.globalOptions && this.globalOptions.experienceKey;
+    this.experienceVersion = this.globalOptions && this.globalOptions.experienceVersion;
```

Comment:
just double checking that globalOptions aren't automatically part of the analytics event?

Reply:
Just checked, and they are, so I'll remove these from here